get-tag

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+set -e
+git describe --tags --abbrev=10 --dirty
```

Comment:
Note that this might suffer from the issue that if you re-run the last master build after tagging it, it will now be run with the _tag_ as the version. You'd need to pass `--long` here and check for `CIRCLE_TAG` to avoid this issue.

Reply:
DOne.